docs/docs/guides/testing.md

```diff
@@ -24,3 +24,78 @@ test('Test multipliedState', () => {
   expect(testSnapshot.getLoadable(multipliedState).valueOrThrow()).toBe(100);
 })
 ```
+
+## Testing Recoil Selectors inside of a React Component
```

Comment:
Your example is probably a better one to start with.  Let's start with it as the beginning of the guide!

Reply:
OK. Will do. Thanks a lot 😄